### PR TITLE
Fix a PHP 8 warning for DCA operations

### PR DESCRIPTION
**Pull request by aschempp**

### CODE REVIEW

**Review by Toflar — APPROVED** [review]

File: core-bundle/src/DataContainer/DataContainerOperation.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function offsetExists(mixed $offset): bool
 
     public function offsetGet(mixed $offset): mixed
     {
-        return $this->operation[$offset];
+        return $this->operation[$offset] ?? null;
```
Comment (leofeyer) [inline core-bundle/src/DataContainer/DataContainerOperation.php:68]: What about line 78? And why is this necessary at all? Did you not call `offsetExists()` in advance?

Comment (aschempp) [reply core-bundle/src/DataContainer/DataContainerOperation.php:68]: as far as I know, `unset()` would not throw a warning, same as `empty()` etc. (because they are not functions but language features). Contao I did not call `offsetExists` (which is actually `isset`), because we never do.

Comment (ausi) [reply core-bundle/src/DataContainer/DataContainerOperation.php:68]: `unset()`, `empty()`, `isset()` and `??` seem to behave exactly the same as an array would, they all check `offsetExists()` first, see https://3v4l.org/YC4it

I think you need to use `??` in the calling code to fix this.

**Comment by ausi** [comment]
How can this be reproduced? Where does the warning appear?

**Comment by aschempp** [comment]
Define a DCA in Contao 5.1 where operations are only strings. 

```
// ...
'operations' => [
    'edit',
    'copy',
    'delete',
    'toggle',
    'show',
]
```

**Comment by fritzmg** [comment]
What's the stack trace for this?

**Comment by ausi** [comment]
> Define a DCA in Contao 5.1 where operations are only strings.

I tried that, but was not able to reproduce the issue.
